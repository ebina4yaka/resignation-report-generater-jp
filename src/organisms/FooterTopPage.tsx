import React, { ReactElement } from 'react'
import Typography from '@material-ui/core/Typography'
import makeStyles from '@material-ui/core/styles/makeStyles'
import Container from '@material-ui/core/Container'
import { createStyles } from '@material-ui/core/styles'
import Copyright from '../atoms/Copyright'

const useStyles = makeStyles((theme) =>
  createStyles({
    root: {
      backgroundColor: theme.palette.background.paper,
      padding: theme.spacing(3, 2),
      marginTop: 'auto',
    },
  })
)

export default function FooterTopPage(): ReactElement {
  const classes = useStyles()
  return (
    <footer className={classes.root}>
      <Container maxWidth="sm">
        <Typography
          variant="body1"
          align="center"
          color="textSecondary"
          component="p"
        >
          Released under the MIT License.
        </Typography>
        <Copyright />
      </Container>
    </footer>
  )
}
