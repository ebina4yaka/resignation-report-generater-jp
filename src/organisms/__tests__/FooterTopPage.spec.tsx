import renderer from 'react-test-renderer'
import FooterTopPage from '../FooterTopPage'

jest.mock('../../atoms/Copyright', () => 'MockedCopyright')

test('Snapshot test for FooterTopPage', () => {
  const tree = renderer.create(<FooterTopPage />).toJSON()
  expect(tree).toMatchSnapshot()
})
