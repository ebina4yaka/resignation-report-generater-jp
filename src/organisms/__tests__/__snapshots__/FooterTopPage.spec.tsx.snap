// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for FooterTopPage 1`] = `
<footer
  className="makeStyles-root-1"
>
  <div
    className="MuiContainer-root MuiContainer-maxWidthSm"
  >
    <p
      className="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-alignCenter"
    >
      Released under the MIT License.
    </p>
    <MockedCopyright />
  </div>
</footer>
`;
